code/__HELPERS/mobs.dm

```diff
@@ -333,7 +333,7 @@
 
 	msg_admin_attack("[key_name_admin(user)] vs [target_info]: [what_done]", loglevel)
 
-/proc/do_mob(mob/user, mob/target, time = 30, uninterruptible = 0, progress = 1, list/extra_checks = list())
+/proc/do_mob(mob/user, mob/target, time = 30, progress = 1, list/extra_checks = list(), only_use_extra_checks = FALSE)
```

Comment:
I checked all the uses of `do_mob` and only the magnetism and lightning spell used `uninterruptible`

Reply:
Might be useful later no?

Reply:
The functionality still remains. Just have to set `only_use_extra_checks` on `TRUE` and leave the `extra_checks` list empty. Or add your own